stages/00_download.py

```diff
@@ -27,16 +27,15 @@ def get_biogrid_html():
     soup = BeautifulSoup(html, features="html.parser")
     return soup
 
-
-tab3_files = re.compile("https://.*/Download/(?!.*OSPREY).*\.tab3.zip$")
+REGEX = "^https://.*/Download/.*/BIOGRID-((?!OSPREY).*\.*(tab3|chemtab|ptm|-4.4.232)|(IDENTIFIERS-4.4.232.tab)).zip$"
```

Comment:
Is there a way we can extract out the version number `4.4.232` to a file? I think a YAML file that just contains `version: 4.4.232` would work well. The scripts that use it can depend on it in `dvc.yaml` (this one and the other).

Also, this is a case for using `re.escape()` because the `.` will be treated a the regex meta-character.